packages/admin/src/containers/Me.container.ts

```diff
@@ -0,0 +1,48 @@
+import { useLazyQuery } from '@apollo/react-hooks';
```

Comment:
Can we make this and the Auth containers part of the `ui` library? Dodn't want to duplicate this logic across projects